src/components/ProjectSettingsTabs/PatchAliasesTab/getFormSchema.ts

```diff
@@ -0,0 +1,68 @@
+import { Field } from "@rjsf/core";
+import { SpruceFormProps } from "components/SpruceForm";
+import widgets from "components/SpruceForm/Widgets";
+import { alias, form } from "../utils";
+import { FormState } from "./types";
+
+const { aliasArraySchema, aliasRowUiSchema } = alias;
+const { overrideRadioBox } = form;
+
+export const getFormSchema = (
+  repoData?: FormState
+): {
+  fields: Record<string, Field>;
+  schema: SpruceFormProps["schema"];
+  uiSchema: SpruceFormProps["uiSchema"];
+} => ({
+  fields: {},
+  schema: {
+    definitions: {
+      aliasArray: aliasArraySchema,
+    },
+    type: "object" as "object",
+    properties: {
+      patchAliases: {
+        title: "Patch Aliases",
+        description:
+          "Specify aliases to use with the CLI. Aliases may be specified multiple times. The result will be their union. All regular expressions must be valid Golang regular expressions. Use an alias with the --alias flag passed to the CLI patch command.",
+        ...overrideRadioBox(
+          "aliases",
+          ["Override Repo Patch Aliases", "Default to Repo Patch Aliases"],
+          {
+            $ref: "#/definitions/aliasArray",
+          }
+        ),
+      },
+    },
+  },
+  uiSchema: {
+    patchAliases: {
+      aliasesOverride: {
```

Comment:
One other thing: we allow users to save the project with Override checked but for aliases we can't actually override unless there are aliases defined so refreshing the page just shows this as the repo values again. Can we disable save unless aliases are actually added?
![image](https://user-images.githubusercontent.com/26798134/154292835-32e29c82-2357-4e6a-8dc4-d43f0f1a1316.png)

Reply:
This relates to the larger issue of "save button is enabled too much," which is tracked by EVG-16268. I'll prioritize that ticket for work soon since this seems to keep coming up.

Reply:
Oh nice. I'll add a comment to that ticket.